package.json

```diff
@@ -49,6 +50,7 @@
     "@semantic-release/github": "^7.2.3",
     "@semantic-release/npm": "^7.1.3",
     "@semantic-release/release-notes-generator": "^9.0.3",
+    "@tsd/typescript": "~4.4.4",
```

Comment:
Note that `@tsd/typescript` will be used to run type tests. It is a dependency of `tsd`, but in `tsd-lite` this is peer dependency. So you can choose any TS version you like.